packages/code-editor/package.json

```diff
@@ -11,7 +11,7 @@
   ],
   "author": "Looker",
   "license": "MIT",
-  "private": true,
+  "private": false,
```

Comment:
@noahamac release-please currently has `code-editor` set to release as `0.1.3` https://github.com/looker-open-source/sdk-codegen/pull/716/files#diff-1e654be0dda2a3ab3237f96fcb7251ecd7367635523a76e1c1302b979354abbcR3

let me know if you want it to bump to `1.0.0` instead and I can tweak the config (and deal with some manual changes required when a dep moves from "pre-1.0.0" to "post-1.0.0")

Reply:
@joeldodge79 code-editor can be bumped to 1.0.0 or left at 0.1.3... Whatever is easier for you